main/main.cpp

```diff
@@ -144,6 +157,7 @@ struct handlers_t
 {
     SipClientT& client;
     ButtonInputHandler<SipClientT, BELL_GPIO_PIN, RING_DURATION_TIMEOUT_MSEC>& button_input_handler;
+	ActuatorHandler<ACTUATOR_GPIO_PIN, false, ACTUATOR_DURATION_TIMEOUT_MSEC>& actuator_handler;
```

Comment:
I think the current design (that is already used before your change) is sub-optimal. Your change shows this.
Adding a actuator (someone who listens to events) should not require this many changes in the main.cpp
It would be better if this could be added in one place (e.g. a container like a tuple) containing all the handlers that should be called in case of an event.

The goal should be to only make it necessary to register the actuator in one line of code, that can be disabled via ifdef if the feature is not necessary.

I will try to come up with something...